testbed/README.md

```diff
@@ -28,6 +28,10 @@ No automatic `Span` propagation between parent and children tasks is provided, a
 
 `TornadoScopeManager` uses a variation of `tornado.stack_context.StackContext` to both store **and** automatically propagate the context from parent coroutines to their children. 
 
+### contextvars
+
+`ContextVarsScopeManager` uses [contextvars](https://docs.python.org/3/library/contextvars.html) module to both store **and** automatically propagate the context from parent coroutines / tasks / scheduled in event loop callbacks to their children.
```

Comment:
IIRC `contextvars` automatically propagates context only when used with `asyncio` - else, the user needs to propagate himself the context (through `contextvars.copy_context()`). Maybe we could add a note here?

Reply:
Do you mean explicit span propagation in multi-threaded code? Because you don't need to do this in single-threaded code, it works implicitly (like `threading.local`). Or I misunderstood your idea?